src/elements/fields/PhoneField/countryData.ts

```diff
@@ -1470,3 +1470,64 @@ export default [
     countryName: '\u00c5land Islands'
   }
 ];
+
+// This list was generated by assessing the differences between two sets
+// Set A consists of countries that Firebase indicates should receive SMS verification without issue (https://firebase.google.com/support/faq/#phone-auth-countries)
+// Set B consists of countries listed above, that are used for libphonenumber-js
+// The list below is the set difference B - A, so those for which libphonenumber-js can parse, but for which Firebase isn't confident in the ability to deliver an SMS
+// The CountryDropdown component also has a safeguard to filter out any countries that it does not support
+export const countryCodesNotWellSupportedByFirebase = [
```

Comment:
would it be better to make this a set and maybe just the country codes supported by firebase? rather than the converse? no need to do an intersection with libphonenumber here since the set can be directly checked in the phone number filtering logic.

also maybe firebaseSMSCountries for brevity?